packages/logging/logging.js

```diff
@@ -44,6 +44,11 @@ Log._intercepted = () => {
 // other process that will be reading its standard output.
 Log.outputFormat = 'json';
 
+// Defaults to true for local development and for backwards compatibility.
+// for cloud environments is interesting to leave it false as most of them have the timestamp in the console.
+// Only works in server with colored-text
+Log.shouldShowTime = true;
```

Comment:
We usually don’t use `should*` for option names. Instead, we prefer imperative names like `showTime` or `enableTime`. That said, using `should*` is completely fine.